fire/docstrings.py

```diff
@@ -177,13 +177,22 @@ def parse(docstring):
   state.returns.lines = []
   state.yields.lines = []
   state.raises.lines = []
+  state.line1 = None
+  state.line1_length = None
+  state.line2_first_word_length = None
+  state.line2_length = None
+  state.line3_first_word_length = None
+  state.max_line_length = 0
 
   for index, line in enumerate(lines):
     has_next = index + 1 < lines_len
     previous_line = lines[index - 1] if index > 0 else None
     next_line = lines[index + 1] if has_next else None
     line_info = _create_line_info(line, next_line, previous_line)
     _consume_line(line_info, state)
+  
+  if state.line2_length:
```

Comment:
This looks like its set up only to apply the heuristic to the final arg encountered, rather than to each arg separately.

Reply:
True. I have addressed this in commit [93ff111](https://github.com/google/python-fire/pull/476/commits/93ff1118f10aea460f761662feea4d6b25f24739)